rss2email/feed.py

```diff
@@ -207,20 +207,25 @@ class Feed (object):
         'digest_post_process',
         ]
 
-    def __init__(self, name=None, url=None, to=None, config=None):
+    def __init__(self, name=None, url=None, to=None, from_email=None, config=None):
         self._set_name(name=name)
         self.reset()
         self.__setstate__(dict(
                 (attr, getattr(self, attr))
                 for attr in self._dynamic_attributes))
         self.load_from_config(config=config)
+        if not from_email is None: self.from_email = from_email
         if url:
             self.url = url
         if to:
             self.to = to
 
     def __str__(self):
-        return '{} ({} -> {})'.format(self.name, self.url, self.to)
+        if self.force_from and\
+           self.from_email != self.config['DEFAULT']['from']:
```

Comment:
If the from field is always set we can also get rid of this special casing.

Reply:
Here, I want to display overrode custom email only if it's different from the one in default.
So I need to do this checks (it's not a None check).

Reply:
Based on the code above, `self.from_email` is going to crash if `from_email` was not given, because then the field doesn’t exist.